src/include/mpir_topo.h

```diff
@@ -5,6 +5,95 @@
  *
  */
 
+/*---------------------------------------------------------------------*/
+/*     (C) Copyright 2017 Parallel Processing Research Laboratory      */
+/*                   Queen's University at Kingston                    */
+/*                Neighborhood Collective Communication                */
+/*                    Seyed Hessamedin Mirsadeghi                      */
+/*---------------------------------------------------------------------*/
```

Comment:
Given the way we do copyrights now (per #4399). Maybe this needs to change? (and all the other places in this PR of course)

Reply:
Yes. :) However, there is no agenda to merge this anytime soon and there are many changes will be needed if we start to consider a merge.

Reply:
No problem. I'm just working my way through the PR backlog in light of the recent discussions on Slack. 😄